source/SpinalHDL/Simulation/simulator_specifics.rst

```diff
@@ -59,6 +59,26 @@ How SpinalHDL simulates the hardware with Synopsys VCS backend
 
 Before using VCS as the simulation backend, make sure that you have checked your system environment as :ref:`VCS environment<vcs_env>`.
 
+How SpinalHDL simulates the hardware with Xilinx XSim backend
+==============================================================
+
+1. SpinalHDL generates a Verilog/VHDL (depended on your choice) hardware model of the DUT.
+2. The HDL model is loaded in the simulator.
+3. The communication between the simulation and the JVM is established through shared memory. The commands are issued to the simulator using XSI.
+
+**Advantages:**
+
+* Support Xilinx built-in primitives and cores.
+
+**Limitations:**
+
+* Xilinx XSim is a **commercial** tool installed with Vivado. It is close source and not free. You have to own the licenses to **legally** use it.
```

Comment:
Your description reads a little shocking.  "You must purchase licenses to legally use this"

... closed source and subject to licensing terms to use.   While development with Xilinx higher performance chips may require a non-free versions of the Vivado tooling suite to target those FPGAs. Many Xilinx based bottom tier development boards (Diligent Arty / Zybo series and equivalent competing products) provide access to obtain a free license for the Vivado suite that is capable of targeting the lower end FPGAs.